util/util.go

```diff
@@ -202,6 +202,15 @@ func PatchClusterCRD(restclient *rest.RESTClient, labelMap map[string]string, ol
 
 }
 
+// GetValueOrDefault checks whether the first value given is set. If it is,
+// that value is returned. If not, the second, default value is returned instead
+func GetValueOrDefault(value, defaultValue string) string {
```

Comment:
Let's call this `GetImagePrefix`

Reply:
I have no strong opinions on renaming, but the intent was to provide a more generic utility function. I could agree that better naming could be had... but, FWIW, my initial suggestion for this naming was more for conveying the purpose/intent than the actual implementation naming.

Reply:
I missed that conversation and just saw it in the email thread.

I understand the genericness of it, but it seems like this is ad-hoc to the imageprefix problem. If other such problems exist, we can certainly apply this to it.

I'll leave it up to @tjmoore4 on what he wants to do. I can go either way.

Reply:
**Short answer**: I'll leave it as it is now, as I think the general case makes sense at this point.

**Longer answer**: While part of me would like to have it named for it's sole current use, the function itself is _much_ more generic than I originally conceived it would be because of the differences in how the prefixes are handled throughout the code. 
If there's any chance that it could be useful for other purposes (and I believe there is), it's now in the correct package (`util`) and named correctly for more general use.
The other option is to rename it now, and change it later if it's used for other purposes, but that adds needless complication in my opinion.
The last, bad, not really an option, option is someone may want to use it, despite it being named `GetImagePrefix` or similar, in which case it would be very confusing and bad.
So, I say best to leave it as is :smile:

Reply:
I'm at peace with this.